source/_integrations/switchbot.markdown

```diff
@@ -166,7 +269,171 @@ cover:
 
 {% endraw %}
 
-## Error codes and troubleshooting
+#### Roller Shade
+The Roller Shade is exposed as a cover entity with control of the position only:
+
+| position | Roller Shade state |
+| -------- | ------------------ |
+| <=20%    | Open               |
+| >20%     | Close              |
+
+Features:
+- open/close/pause
+- set position
+- get position
+- get battery level
+
+### Sensors
+
+Sensor entiteis are added for thermometer and hygrometer devices, motion sensor, contact sensor, leak sensor, and remote button.
+
+#### Meter
+
+Features:
+- get temperature
+- get humidity
+- get battery level
+
+#### Meter Plus
+
+Features:
+- get temperature
+- get humidity
+- get battery level
+
+#### Indoor/Outdoor Meter
+
+Features:
+- get temperature
+- get humidity
+- get battery level
+
+#### Meter Pro
+
+Features:
+- get temperature
+- get humidity
+- get battery level
+
+#### Meter Pro CO2 Monitor
+
+Features:
+- get temperature
+- get humidity
+- get carbon dioxide
+- get battery level
+
+#### Contact Sensor
+
+Features:
+
+- open or closed state
+- motion detection state
+- get battery level
+
+#### Motion Sensor
+
+Features:
+
+- motion detection state
+- get battery level
+#### Water Leak Detector
+This is an encrypted device.
+
+Features:
+- leak or dry state
+- get battery level
+
+#### Remote
+
+Features:
+- get battery level
+
+### Lights
+
+Light entities are added for Color Bulb and LED Strip Light.
+
+#### Color Bulb
+
+Features:
+- turn on or off
+- change brightness
+- change color temperature
+- change color
+
+#### LED Strip Light
+
+Features:
+- turn on or off
+- change brightness
+- change color
+
+### Locks
+
+Note: The integration currently only uses the primary lock state; in dual lock mode, not all things might work properly.
+
+#### Lock
+
+This is an encrypted device.
+
+Features:
+- Lock or unlock
+- open or closed state
+- auto-lock paused state
+- calibration state
+- get battery level
+
+#### Lock Pro
+
+This is an encrypted device.
+
+Features:
+
+- Lock or unlock
+- open or closed state
+- auto-lock paused state
+- calibration state
+- get battery level
+
+### Hubs
+
+Some of the hubs can be served as a bridge while the sensor data can be retrieved via Bluetooth connection. Hub 2 displays temperature and humidity through a sensor cable. Without a digital display, Hub Mini Matter Enabled can also read from a sensor cable.
+
+#### Hub 2
+
+Features:
+
+- get temperature
+- get humidity
+- get light level
+
+#### Hub Mini Matter Enabled
+
+Features:
+
+- get temperature
+- get humidity
+
+## Data updates
+
+Devices are polled for data updates every 5 seconds. When you make changes through Home Assistant (e.g., switching a device on), the device's state is updated immediately rather than waiting for the next poll.
```

Comment:
```suggestion
Devices are {% term polling polled %} for data updates every 5 seconds. When you make changes through Home Assistant (for example by switching a device on), the device's state is updated immediately rather than waiting for the next poll.
```